.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,16 @@
 ## Purpose:
 What issue(s) does your PR address?
 
-## Results
-If applicable, briefly report your findings.
+## Strategy
+What was your strategy for this new or edited analysis?
+
+## Concerns/Questions for reviewers:
+What things should reviewers look out for?
 
 ## Pull Request Check List:
-* [ ] Run a linter
-* [ ] Set the seed (if applicable)
-* [ ] Comments and/or documentation up to date
```

Comment:
We should probably keep this checkpoint re comments and documentation being up to date in here

Reply:
Okay. Will add back.